flowermd/base/molecule.py

```diff
@@ -513,14 +520,20 @@ def monomer(self):
         return self._mb_molecule
 
     def _build(self, length):
+        if self.periodic_bond_axis:
+            add_hydrogens = False
+        else:
+            add_hydrogens = True
         chain = mbPolymer()
         chain.add_monomer(
             self.monomer,
             indices=self.bond_indices,
             separation=self.bond_length,
             orientation=self.bond_orientation,
         )
-        chain.build(n=length, sequence="A")
+        chain.build(n=length, sequence="A", add_hydrogens=add_hydrogens)
+        if self.periodic_bond_axis:
+            chain.create_periodic_bond(axis=self.periodic_bond_axis)
```

Comment:
Maybe we can run a quick validation on `self.periodic_bond_axis` to make sure its value is valid (i.e. "x", "y", or "z") before calling the periodic bond function.

Reply:
Good idea, this should be in there now.